src/test/java/com/jezhumble/javasysmon/ProcessTreeFunctionalTest.java

```diff
@@ -8,9 +8,8 @@ public void testShouldKillChildProcesses() {
         try {
             JavaSysMon monitor = new JavaSysMon();
             Assert.assertEquals(0, monitor.processTree().find(monitor.currentPid()).children().size());
-            Runtime.getRuntime().exec("ant sleep");
```

Comment:
Given it's extremely rare to find someone with `ant` installed, I thought it would be okay to remove this from the tests, and do the assertion based only on the sleep command.